pipeline/outputs/cloudwatch.md

```diff
@@ -44,6 +46,49 @@ In your main configuration file append the following _Output_ section:
     auto_create_group On
 ```
 
+For using the `mem` input plugin and sending memory usage metrics to CloudWatch, we can consider the following example config file. Here, we use the `aws` filter which adds `ec2_instance_id` and `az` (availability zone) to the log records. Later, in the output config section, we set `ec2_instance_id` as our metric dimension.
```

Comment:
Put a heading above this and title it "metrics tutorial"

Reply:
Also note that right now, only CPU and Mem metrics can be sent

Reply:
Also note that this is different than the log_format field which can be used if the incoming records are already formatted as EMF- explain the difference between the two features.

Reply:
Updated